packages/client/components/NewMeetingTeamPicker.tsx

```diff
@@ -32,13 +33,36 @@ const NewMeetingTeamPicker = (props: Props) => {
       isDropdown: true
     }
   )
+
+  const selectedTeam = useFragment(
+    graphql`
+      fragment NewMeetingTeamPicker_selectedTeam on Team {
+        ...NewMeetingTeamPickerAvatars_team
+        name
+      }
+    `,
+    selectedTeamRef
+  )
+
+  const teams = useFragment(
+    graphql`
+      fragment NewMeetingTeamPicker_teams on Team @relay(plural: true) {
```

Comment:
TIL, Good to know the [`@relay(plural: Boolean)` GraphQL directive](https://relay.dev/docs/api-reference/graphql-and-directives/#relayplural-boolean)